Tests/AmplitudeTests/AmplitudeTests.swift

```diff
@@ -82,6 +82,29 @@ final class AmplitudeTests: XCTestCase {
         XCTAssertEqual(lastEvent?.language!.isEmpty, false)
     }
 
+    func testFilterAndEnrichmentPlugin() {
+        let apiKey = "testFilterAndEnrichmentPlugin"
+        let enrichedEventType = "Enriched Event"
+        storageTest = TestPersistentStorage(storagePrefix: "storage-\(apiKey)")
+        let amplitude = Amplitude(configuration: Configuration(
+            apiKey: apiKey,
+            storageProvider: storageTest
+        ))
+
+        let testPlugin = TestFilterAndEnrichmentPlugin()
```

Comment:
nit: I think it might make it more clear what is being filtered if you define the filter plugin inline to the test vs TestUtilities. I don't think that the filter plugin will be used in other tests.